ntbs-service/Models/Entities/NotificationDisplay.cs

```diff
@@ -96,7 +98,33 @@ private bool GetIsLastLinkedNotificationOverOneYearOld()
             {
                 yearDiff -= 1;
             }
+
             return yearDiff;
         }
+
+        private int SiteOrdering(Site site)
+        {
+            switch (site.SiteId)
+            {
+                case 1: return 1;
+                case 12: return 2;
+                case 13: return 3;
+                case 2: return 4;
+                case 3: return 5;
+                case 4: return 6;
+                case 5: return 7;
+                case 10: return 8;
+                case 11: return 9;
+                case 7: return 10;
+                case 8: return 11;
+                case 9: return 12;
+                case 6: return 13;
+                case 14: return 14;
+                case 15: return 15;
+                case 16: return 16;
+                case 17: return 17;
+                default: return 18;
```

Comment:
Is this the best place to be putting this, or should this go in the database (that perhaps could get reused by reporting)?

Reply:
Yeah can be put into the database, will just mean any changes will have to use a migration - though I can't imagine this will be changing much

Reply:
@LeoCie Are you planning to update reporting to use the ordering from the NTBS database?

Reply:
@JoshCadney yes, but I will need the dacpac from azure so I'll wait until this is in int